src/dialogsformaction.cpp

```diff
@@ -27,7 +27,7 @@ void DialogsFormAction::init()
 {
 	set_keymap_hints();
 
-	f.run(-3); // compute all widget dimensions
```

Comment:
Out of curiosity: why all the methods in this PR are called through `this`? I know C++ sometimes needs this when the compiler can't figure out where the name comes from, but most of the time it should be okay without. Is it just for clarity, to visually distinguish methods from functions?

(I'm generally fine with running experiments on Newsboat's codebase, so if you're doing this just to see how it goes, it's okay.)

Reply:
> Out of curiosity: why all the methods in this PR are called through this?

I guess I added it for clarity.
On rereading, I'm not even sure it improves anything. I removed all of the added `this->` parts for consistency with existing code.